docs/tutorials/kubectl/app-management-overview.md

```diff
@@ -0,0 +1,238 @@
+---
+title: App Management Overview
+---
+
+{% capture overview %}
+
+*Kubectl* supports several different approaches for creating and managing
+applications in a Kubernetes cluster.  This document provides an overview
+of the different approaches and which environment each approach is best
+suited towards.
+
+{% endcapture %}
+
+{% capture prerequisites %}
+
+{% include task-tutorial-prereqs.md %}
+
+### Prerequisites
+
+- [Kubernetes Basics Tutorial](/docs/tutorials/kubernetes-basics/)
+
+{% endcapture %}
+
+{% capture objectives %}
+
+- Explore the different approaches for managing applications supported by *kubectl*.
+- Explore which environment each approach is best suited for.
+
+{% endcapture %}
+
+
+{% capture lessoncontent %}
+
+# App Management Styles Table
+
+**Currently, a given resource should be managed using only 1 style.  Attempting
+to mix-and-match styles for the same resource will result in undefined behavior.**
+
+| Management Style          | Environment             | Writers  | Operates Best On     | Learning Curve |
+|---------------------------|-------------------------|----------|----------------------|----------------|
+| Imperative Commands       | Dev or Small Clusters   | 1+       | Live Resources       | Lowest         |
+| Basic Yaml Config         | Production Clusters     | 1        | Individual Files     | Moderate       |
```

Comment:
Using the term `Config` here is a bit confusing for me.  This is all Config (and config is way way overloaded).  

Consider going with "Imperative/Declarative" or let's find another way to slice this.

Perhaps another way to think about this would be where the ultimate source of desired state is.

* *Kubernetes Managed* - this is imperative and the final source of truth is the API server.
* *YAML only* - here some YAML files (ideally stored in SCM) are the only source of truth
* *Merged* - here there are multiple agents modifying the resource (give examples) and the system smartly merges configuration